Apps/Database/src/Migrations/20200528223039_PhoneBugfixes.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace HealthGateway.Database.Migrations
```

Comment:
This migration will result in the loss of information, it should be an alter column instead of drop.  I would prefer if you could remove the migration and reset the model then try making the rename change first and verify if the migration created is an alter.